documentation/decisions/adr/000n-ticket-tracking.md

```diff
@@ -0,0 +1,199 @@
+# Ticket Tracking
+
+- **Status:** Accepted <!-- REQUIRED -->
+- **Last Modified:** 2023-07-11 <!-- REQUIRED -->
+- **Related Issue:** [#98](https://github.com/HHS/grants-api/issues/98) <!-- RECOMMENDED -->
+- **Deciders:** Lucas Brown, Esther Oke, Sumi Thaiveettil, Sarah Knopp, Aaron Couch, Billy Daly 
+- **Tags:** communications, sprint planning, agile <!-- OPTIONAL -->
+
+## Context and Problem Statement
+
+The project needs a system for tracking ongoing development work within the project, preferably as a series of tickets that can be organized into sprints. This system would both enable internal stakeholders to prioritize key tasks and assignments throughout the project and help communicate those priorities to external stakeholders.
+
+The goal of this ADR is to evaluate a series of ticket tracking systems and select the one we will be using for the project.
+
+## Decision Drivers <!-- RECOMMENDED -->
+
+#### Must Have
+
+- **Ticket Tracking:** Tickets can be organized into sprints and tracked as part of larger milestones or epics
+- **Public Access:** Without logging in, members of the public can see tickets that are being worked on
+- **Public Requests:** Members of the public can submit bug reports and feature requests and track how that work is being prioritized
+- **Templates:** The system supports default templates for different types of tickets which prompts the person creating the ticket for a specific set of information
+- **Authority to Operate (ATO):** The platform already must be authorized under the Grants.gov ATO (Authority to Operate) or ATO coverage must be requested
+
```

Comment:
Add "- **Reporting:** Users can report on key metrics like burndown, point allocation, etc. from directly within the tool
"

Reply:
Good point!